cisco_aci/datadog_checks/cisco_aci/helpers.py

```diff
@@ -0,0 +1,161 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import re
+
+
+def parse_capacity_tags(dn):
+    tags = []
+    pod = get_pod_from_dn(dn)
+    if pod:
+        tags.append("fabric_pod_id:{0}".format(pod))
+    node = get_node_from_dn(dn)
+    if node:
+        tags.append("node_id:{0}".format(node))
+
+    return tags
+
+
+def get_pod_from_dn(dn):
+    pod = re.search('pod-([0-9]+)', dn)
```

Comment:
we could compile these regex once, at the module level

Reply:
That is likely a good idea, is the performance impact that big?

Reply:
Python caches REs, but yes.